pkg/linter/lintRuleNonRelevantCIDRCommon.go

```diff
@@ -25,7 +25,7 @@ type ruleNonRelevantCIDR struct {
 // functionality used by both SG and NACL lints
 ////////////////////////////////////////////////////////////////////////////////////////////
 
-func findRuleNonRelevantCIDR(configs map[string]*vpcmodel.VPCConfig, filterLayerName string) (res []ruleNonRelevantCIDR, err error) {
+func findRuleNonRelevantCIDR(configs map[string]*vpcmodel.VPCConfig, filterLayerName string) (res []finding, err error) {
```

Comment:
Why this change?

Reply:
all the checkForFilter function must have the same signeture